notebooks/Emukit-multi-fidelity-bayesian-optimization.ipynb

```diff
@@ -0,0 +1,692 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Multi-Fidelity Bayesian Optimization\n",
+    "\n",
+    "This notebook contains an example of using Emukit to perform Bayesian optimization on a function where low-fidelity approximations are availalbe. \n",
+    "\n",
+    "The high fidelity function to be optimized is the forreseter function given by: \n",
+    "\n",
+    "$$\n",
+    "f_{high}(x) = (6x - 2)^2 \\sin(12x - 4)\n",
+    "$$\n",
+    "\n",
+    "The low fidelity approximation of this function is given by:\n",
+    "\n",
+    "$$\n",
+    "f_{low}(x) = 0.5 f_{high}(x) + 10 (x - 0.5) + 5\n",
+    "$$\n",
+    "\n",
+    "Both are defined in the interval [0, 1].\n",
+    "\n",
+    "The high fidelity function is assigned a cost of $3$, whereas the low fidelity function is assigned a cost of $1$.\n",
```

Comment:
"is assigned a cost of X" is a bit unclear. "Each single evaluation of f costs X" would be clearer to me

Reply:
Changed

Reply:
Is it?

Reply:
Weird, I'm sure I changed them in the github editor. I'll try again